features/behave.ini

```diff
@@ -0,0 +1,5 @@
+[behave.userdata]
+fixtures_project = /app/fixtures/small
+hasura_endpoint = http://graphql-engine:8080
+graphql_endpoint = v1/graphql/
+shopozor_project = /db/
```

Comment:
Those parameters can also be provided through the command-line to `behave`. I find it more practical to have them here.